activerecord/lib/rails/generators/active_record/application_record/application_record_generator.rb

```diff
@@ -16,10 +16,10 @@ def create_application_record
 
         def application_record_file_name
           @application_record_file_name ||= if namespaced?
-                                              "app/models/#{namespaced_path}/application_record.rb"
-                                            else
-                                              "app/models/application_record.rb"
-                                            end
+            "app/models/#{namespaced_path}/application_record.rb"
+          else
+            "app/models/application_record.rb"
+          end
```

Comment:
Unfortunately code climate runs a version of rubocop older than the one our config uses. This is why you fixed locally and broke in code climate.